R/animint.R

```diff
@@ -1022,69 +1037,89 @@ getCommonChunk <- function(built, chunk.vars, aes.list){
       built[, col.name] <- paste(built[, col.name])
     }
   }
-  built.by.chunk <- split(built, built[, chunk.vars], drop = TRUE)
```

Comment:
i removed this split by all chunk variables, in order to speed up the common chunk computation